test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -102,11 +107,40 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		</java>
 	</target>
 
-	<target name="TCK.generate.dump">
+	<target name="TCK.initialize.cache" description="Destroy the old cache and create a new one">
+		<echo>Initializing cache</echo>
+		<!-- Initialize the cache so that addtestjithints option adds jithints to the shared cache in TCK.generate.dump-->
+		<echo>Running j9vm.test.corehelper.StackMapCoreGenerator to create a new cache</echo>
+		<echo>Using JVM : ${sdk.executable}${sdk.executable.ext}</echo>
+		<echo>classname = "j9vm.test.corehelper.StackMapCoreGenerator"</echo>
+		<echo>Java VM Args:</echo>
+		<echo>	jvmarg = -Xshareclasses:name=ddrextjunitSCC,reset,modified=mod1</echo>
+		<echo>
+		</echo>
+		<java fork="true" jvm="${JAVA_COMMAND}" classname="j9vm.test.corehelper.StackMapCoreGenerator" timeout="1200000" failonerror="true">
+			<jvmarg value="-Xshareclasses:name=ddrextjunitSCC,reset,modified=mod1" />
+			<jvmarg value="-Xitsn1000" />
+			
+			<classpath refid="tck.class.path" />
+		</java>
+	</target>
+
+	<target name="TCK.generate.dump" depends="TCK.initialize.cache">
 		<echo>Generate dump file</echo>
+		<echo>JAVA_COMMAND: ${JAVA_COMMAND}</echo>
+		<echo>DUMP_OPTION: ${DUMP_OPTION}</echo>
+		<echo>EXTRADUMPOPT: ${EXTRADUMPOPT}</echo>
+
 		<exec executable="${JAVA_COMMAND}" failonerror="true">
 			<arg line="${DUMP_OPTION}" />
+			<arg line="-Xitsn1000" />
+			<arg line="${EXTRADUMPOPT}" />
+			<arg value="-Xjit:{j9vm/test/corehelper/CoreGen.main*}(count=0)"/>
```

Comment:
Is it possible to use `${EXTRADUMPOPT}` for passing in these two options (-Xitsn1000 and -Xjit...)?

Reply:
I don't think that's worth it: we'd have to be careful about quoting or otherwise ensuring that `<arg line=...>` properly separates arguments - a problem that doesn't arise with `<arg value=...>`.

Reply:
Yes, quoting and special characters may be a problem.